bluetooth/src/iosMain/kotlin/scanner/Scanner.kt

```diff
@@ -48,12 +48,13 @@ import platform.darwin.dispatch_get_main_queue
 actual class Scanner internal constructor(
     permissions: Permissions,
     private val connectionSettings: ConnectionSettings,
+    private val scanSettings: Map<Any?, *>?,
```

Comment:
This map says nothing about what settings are possible to be. I know it's flexible but you have to look into code and find place manually to answer that question. Here is two options. Add comments, reference what key are acceptable, etc. Or introduce custom enum class with possible settings (for now just allow duplicates in scan) so it will be more strict and obvious how to use (preferred one).

Reply:
Yeah this is a little too "Javascript", use types

Reply:
Indeed, I made it more friendly here [f8dd881](https://github.com/splendo/kaluga/pull/501/commits/f8dd881894edaae59a818c1479eaecb24af1a5ff). Btw there are very limited options for [iOS](https://developer.apple.com/documentation/corebluetooth/cbcentralmanager/peripheral_scanning_options).